source/crud/crud.md

```diff
@@ -2496,6 +2513,8 @@ aforementioned allowance in the SemVer spec.
 
 ## Changelog
 
+- 2024-11-05: Define `findOne` operation as optional.
```

Comment:
You can also note that you're adding guidance on `limit` and `batchSize` for `find` operations.